avo-hq/avo
# feature: card visibility

# Description


Fixes https://github.com/avo-hq/avo/issues/1604
Docs: https://docs.avohq.io/2.0/dashboards.html#cards-visibility
Video: https://www.youtube.com/watch?v=2LpaP8UdzxY



# Checklist:




## Manual review steps


1. Go to Dashy dashboard
1. Change a card `visible` to false. Observe it is hidden
2. change it to true. Observe it is visible.

Manual reviewer: please leave a comment with output from the test if that's the case.
